stripe/src/main/java/com/stripe/android/view/PaymentMethodsActivity.kt

```diff
@@ -63,7 +63,6 @@ class PaymentMethodsActivity : AppCompatActivity() {
         PaymentMethodsAdapter(
             args,
             addableTypes = args.paymentMethodTypes,
-            initiallySelectedPaymentMethodId = viewModel.selectedPaymentMethodId,
```

Comment:
revert this so that the correct payment method id is selected when opening `PaymentMethodsActivity`

Reply:
oh. missed that one, thanks!